geocity/apps/reports/models.py

```diff
@@ -650,40 +651,72 @@ class Meta:
         verbose_name = _("Adresse de facturation")
 
 
+CONTACT_TYPE_AUTHOR = 999
+CONTACT_TYPE_CHOICES_RECIPIENT = CONTACT_TYPE_CHOICES + (
+    (CONTACT_TYPE_AUTHOR, _("Author")),
+)
+
+
 class SectionRecipient(Section):
     padding_top = padding_top_field(default=40)
     is_recommended = models.BooleanField(
         _("Recommandée"),
         default=False,
         help_text=_('Ajoute le texte "RECOMMANDEE" en première ligne'),
     )
-    uses_dynamic_recipient = models.BooleanField(
-        _("Destinataire dynamique"),
-        default=True,
+    principal_recipient = models.PositiveSmallIntegerField(
+        _("Destinataire principal"),
+        choices=CONTACT_TYPE_CHOICES_RECIPIENT,
+        default=CONTACT_TYPE_AUTHOR,
+        help_text=_(
+            "Utilisé par défaut. Si celui-ci n'existe pas, prend le destinataire secondaire"
```

Comment:
There will always be an author and thus a principal recipient, isn't it ?

Reply:
If this isn't for anonymous submissions, yes